adapters/postgres/postgres.go

```diff
@@ -113,6 +113,33 @@ func WhereByRequest(r *http.Request, initialPlaceholderID int) (whereSyntax stri
 	return
 }
 
+// JoinByRequest implements join in queries
+func JoinByRequest(r *http.Request) (values []string, err error) {
+	joinValues := []string{}
+
+	u, _ := url.Parse(r.URL.String())
+	joinStatements := u.Query()["_join"]
+
+	for _, j := range joinStatements {
+		joinArgs := strings.Split(j, ":")
```

Comment:
@henriquechehad so could please change to call check just after query is completed?